rusty_pizza_server/src/util/money.rs

```diff
@@ -43,6 +43,14 @@ impl Add for Money {
     }
 }
 
+impl AddAssign for Money {
+    fn add_assign(&mut self, other: Self) {
+        *self = Self {
```

Comment:
For my understanding:
You overwrite the old `Money` object with a new one using a pointer?
Why are you not using somthing like `self.cents = self.cents + other.cents`

Reply:
Yes. This has two reasons:
1. The [official documentation](https://doc.rust-lang.org/std/ops/trait.AddAssign.html) shows an example doing it exactly this way.
2. `Money` is designed to be immutable. Overwriting the number of cents would break this immutability.